app/routes/docs_.schema.$version.$/components.tsx

```diff
@@ -36,9 +36,16 @@ export type Schema = {
   required?: string[]
 }
 
+function FormatLinkString(path: string) {
```

Comment:
```suggestion
function formatLinkString(path: string) {
```

Function names generally start with a lowercase letter.